paasta_itests/paasta_serviceinit.feature

```diff
@@ -44,6 +44,14 @@ Feature: paasta_serviceinit
       And we wait for it to be deployed
      Then paasta_serviceinit status -vv for the service_instance "test-service.main" exits with return code 0 and the correct output
 
+  Scenario: paasta_serviceinit can run status -s service -i instances
+    Given a working paasta cluster
+      And I have yelpsoa-configs for the marathon job "test-service.main"
+      And we have a deployments.json for the service "test-service" with enabled instance "main"
+     When we run the marathon app "test-service.main"
+      And we wait for it to be deployed
+     Then paasta_serviceinit status -s "test-service" -i "main,test" has the correct output for instance main and exits with non-zero return code for instance test
```

Comment:
How about just put this on line 46? It looks like we don't need the whole scenario